RELEASE.md

```diff
@@ -13,20 +13,27 @@ The **Open Quantum Safe (OQS) project** has the goal of developing and prototypi
 Release notes
 =============
 
-This is the 2023-10 snapshot release of OQS-OpenSSH, released on October 21, 2023. This release is intended to be used with liboqs version 0.9.0.
+This is the 2024-08 snapshot release of OQS-OpenSSH, released on August 30, 2024. This release is intended to be used with liboqs version 0.10.1.
 
 What's New
 ----------
 
-This is the seventh snapshot release of the OQS fork of OpenSSH.  It is based on OpenSSH 8.9 portable 1.
+This is the eighth snapshot release of the OQS fork of OpenSSH.  It is based on OpenSSH 9.7 portable 1.
 
-- Update algorithm list in line with `liboqs` v0.9.0.
+- Updated fork to track upstream OpenSSH 9.7.
+- Update algorithm list in line with `liboqs` v0.10.1.
+  + Introduces generic support for ML-KEM, ML-DSA, SNTRUP, BIKEr4, MAYO, and Falcon (Padded).
+- Added support for x25519 hybrid key exchange algorithms.
+  + Working interop support for `x25519-kyber-512r3-sha256-d00@amazon.com` key exchange.
+  + Working interop support for `sntrup761x25519-sha512@openssh.com` key exchange.
+- Support for all ML-KEM based hybrid key exchanges in https://datatracker.ietf.org/doc/draft-kampanakis-curdle-ssh-pq-ke/
```

Comment:
Some comment here that these are ipd.

Reply:
There should be no reference to -ipd any more. We agreed yesterday to drop it. We introduced the notion of aliases exactly to avoid this from happening (oqs-provider has no notion of these very ephemeral alg names).

Reply:
But the release notes say that this is meant to be used with liboqs 0.10.1 and has a target release date of August 30, which contains -ipd, not the final version.

Reply:
Two thoughts on that: 1) We have introduced the notion of algorithm aliases. oqsprovider uses that to completely hide the notion of -ipd. My argument is that openssh could (should!) do the same (hide an algorithm that effectively isn't). 2) Why do we need to do a release of oqs-openssh now, particularly if this would force (if not using the alias names) to explicitly _newly_ expose -ipd --- that we just agreed to _drop_? This IMO makes no sense. If @geedo0 indeed has to leave the project after his vacation again I volunteer to do a fully NIST-spec aligned release of oqs-openssh (i.e., after 0.11.0 is out, i.e., with -ipd dropped).

My proposal is to change this text to read "aligned with liboqs main" and do a release after 0.11.0.

Reply:
As @baentsch has alluded to, we can kind of have it both ways. I have coded this up using the alias non-IPD aliases. We've also [documented](https://github.com/open-quantum-safe/openssh/blob/OQS-v9/README.md?plain=1#L91) that this algorithm support will follow whatever the underlying ML-KEM implementation uses. Assuming some level of ambiguity with what ML-KEM means, the release notes and README taken together are accurate.

> My proposal is to change [correction] the README text to read "aligned with liboqs main" and do a release after 0.11.0.

That's how it reads today thanks to your [initial insistence](https://github.com/open-quantum-safe/openssh/pull/165#pullrequestreview-2236258758) 😄

I'll add a note to the RELEASE.md going over these details. I don't think we need to hold off the release over this detail.

Reply:
Okay, thanks!